packages/network/src/network.js

```diff
@@ -14,6 +15,125 @@ import '@agoric/store/exported.js';
  */
 export const ENDPOINT_SEPARATOR = '/';
 
+const Shape1 = /** @type {const} */ ({
+  /**
+   * Data is string | Buffer | ArrayBuffer
+   * but only string is passable
+   */
+  Data: M.string(),
+  Bytes: M.string(),
+  Endpoint: M.string(),
+  // XXX avoid: cannot check unrecognized tag "Vow": "[Vow]"
```

Comment:
Should this comment make it into checked-in code?

Reply:
it's intended to, yes. `XXX` mark

> XXX acknowledges some tech debt. It's not broken and doesn't need to be tracked as something to change, but it should not be emulated.
 -- https://github.com/Agoric/agoric-sdk/wiki/Coding-Style

it could perhaps be a TODO and cite the relevant issue, but we haven't filed one. @erights made a PR, though:

 - https://github.com/endojs/endo/pull/2091

I guess TODO is better.